curation_concerns-models/app/jobs/create_derivatives_job.rb

```diff
@@ -1,14 +0,0 @@
-class CreateDerivativesJob < ActiveFedoraIdBasedJob
```

Comment:
Why did you delete this file?  It's in both sufia/pcdm and curation_concerns/master.